README.md

```diff
@@ -0,0 +1,45 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This project is an introduction to the github actions and linters.
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+    clone my code by executing :
+    git clone : https://github.com/maaloum/Hello-Microverse.git
+
+## Author
+
+👤 **Maaloum**
+
+- GitHub: [@gMaaloum](https://github.com/maaloum/)
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/maaloum/Hello-Microverse/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thank you Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] - Kindly make sure to include an MIT.md in your project. If you may be in doubt, have a look at [this](https://github.com/microverseinc/readme-template/blob/master/MIT.md) example MIT.md this way your documentation will be 100% complete :sunglasses: .